cmd/plugin.go

```diff
@@ -209,26 +223,50 @@ var (
 
 	enableCommand = &cobra.Command{
 		Use:     "enable",
-		Short:   "Enabled or Disable plugin",
-		Args:    cobra.ExactArgs(2), // TODO: Can I check real git repo?
-		Example: "vatz plugin enable <pluginName> <true/false>",
+		Short:   "Enable plugin",
+		Example: "vatz plugin enable --plugin <pluginName>",
 		PreRunE: func(cmd *cobra.Command, args []string) error {
 			_, err := config.InitConfig(configFile)
 			return err
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			pluginName := viper.GetString("enable_plugin")
 			pluginDir, err := config.GetConfig().Vatz.AbsoluteHomePath()
 			if err != nil {
 				return err
 			}
 
-			log.Debug().Str("module", "plugin").Msgf("enable installed plugin %s at %s", args[0], pluginDir)
+			log.Debug().Str("module", "plugin").Msgf("enable installed plugin %s at %s", pluginName, pluginDir)
+
+			mgr := plugin.NewManager(pluginDir)
+			err = mgr.Update(pluginName, true)
```

Comment:
The name `Update` seems quite ambiguous, what do you think `SetEnable` for the new one?(but I'm not sure doing this in this PR is a good choice.)

Reply:
@rootwarp 
fixed!!